apps/omg_watcher_info/lib/omg_watcher_info/db/traceable_repo.ex

```diff
@@ -0,0 +1,103 @@
+# Copyright 2019-2020 OmiseGO Pte Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+defmodule OMG.WatcherInfo.DB.TraceableRepo do
```

Comment:
It's a workaround for https://github.com/spandex-project/spandex_ecto/issues/22. I tried using `span` and `trace` decorators. They don't record traces from queries

Reply:
even though this works, great job @ayrat555! I think it's super intrusive to our elixir-omg Repo. well... you had to wrap the whole Repo module to make this work.

what if we fork spandex_ecto and bake in the solution?

Reply:
yeah, I talked with @achiurizo a couple of days ago. He suggested the same. I'll look into it

Reply:
wonderful!

Reply:
@InoMurko pr is ready for review